### ISSUE TITLE
mastering git skill level B1

### ISSUE THREAD

**Issue by HannesSommer** [ethz-asl/tutorial_ideas#11]
It seems to me that for some basic users of git at the lab it could be pretty useful or at least relaxing to understand ```git``` conceptually just a tiny bit better.

So I would like to try to give a *B1* tutorial for those who like to become a bit less afraid and more fluent in ```git```.

This is again an experiment and *without any warranty*. As preparation I will talk to git advanced and basic users (both easy to find at the lab) to find out what could be the most useful to do for the current basic users within 1h dedicated to ```git```, of course aiming at being beneficial for the everyday git work.

This is _not_ aiming at providing best practices like #4. It is much more to learn the means to cope with unexpected situations - without panicking ;) and to become more efficient with the usual tasks and maybe eventually find your own best practice.

The first experiment will take place this 

**Thursday (26.2.15) at 11:00**. 

(Yes, that is the planning reading group time slot). It can be repeated if there is still enough interest.

For the interested:
Please feel free to write down here (as comments) ```git``` aspects / actions you would like to understand or learn better or what you think is useful to know. But please mind the target scope ;).

For the experts: 
I would of course appreciate any kind of input.

**Comment by mikebosse** [comment]
- git subtree merge
- aborting, reseting git merge
- git rebase vs git merge

**Comment by HannesSommer** [comment]
Thanks for the input! I'm afraid I would count ```git subtree *``` is clearly C* level and will not find a place in this tut - or did you mean something else? The others will certainly play a role :).

**Comment by simonlynen** [comment]
@HannesSommer I think git bisect is a tool that you should at least mention. Many people are not aware of how much it can simplify your life when searching for a bug.

**Comment by HannesSommer** [comment]
Yes, thanks. This is also a good and welcome argument to keep or make your commits compile ;).
